packages/commonwealth/client/scripts/views/components/sidebar/CommunitySection/CommunitySection.tsx

```diff
@@ -89,11 +127,22 @@ export const CommunitySection = ({ showSkeleton }: CommunitySectionProps) => {
   return (
     <>
       <div className="community-menu">
+        <ProfileCard />
         {user.isLoggedIn && (
           <>
             <AccountConnectionIndicator
               connected={!!user.activeAccount}
               address={user.activeAccount?.address || ''}
+              onAuthModalOpen={(modalType) =>
+                setAuthModalType(modalType || AuthModalType.SignIn)
```

Comment:
the auth modal should also be provided with the `showWalletsFor` param, and it should show wallets for the active community.